js/index.js

```diff
@@ -1,6 +1,70 @@
 // Iteration 1: Names and Input
 //
+
+const hacker1 = "Matthew"
+const hacker2 = "Joao"
+
+console.log (`The driver's name is ${hacker1}`);
+console.log (`The navigator's name is ${hacker2}`);
+
+
+
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length){
+  console.log (`The driver has the longest name, it has ${hacker1.length} characters.`)
+} else if (hacker1.length < hacker2.length)  {
+  console.log (`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+} else {
+  console.log (`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
 
 // Iteration 3: Loops
+
+// Iteration 3.1
+let driver = '';
+
+for (let i = 0; i <= hacker1.length; i++) {
+    driver += hacker1[i].toUpperCase() + ' ';
+    console.log(driver)
+}
+
+// Iteration 3.2
+
+let navigator = '';
+for (let i = hacker2.length - 1; i >= 0; i--) {
+    navigator += hacker2[i];
+
+    console.log(navigator)
```

Comment:
If the `console.log` was outside the loop it will just print on time and the final result ^^